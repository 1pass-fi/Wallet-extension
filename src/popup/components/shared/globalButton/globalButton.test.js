import '@babel/polyfill'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'

import GlobalButton from '.'

describe('Test for GlobalButton component', () => {
  describe('Type send', () => {
    it('renders correctly', () => {
      const { container } = render(<GlobalButton currency='KOI' type='send'/>)
      expect(container).toMatchSnapshot('send-button')
    })
  })

  describe('Type lock', () => {
    it('renders correctly', () => {
      const { container } = render(<GlobalButton type='lock'/>)
      expect(container).toMatchSnapshot('send-button')
    })
  })
})
