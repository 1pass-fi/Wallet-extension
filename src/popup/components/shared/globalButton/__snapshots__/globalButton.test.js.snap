// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for GlobalButton component Render without crashing renders correctly 1`] = `
<div>
  <div>
    <button
      class="global-button"
    >
      <div
        class="global-button-content"
      >
        <div
          class="global-button-icon"
        >
          <svg />
        </div>
        <div
          class="global-button-label"
        >
          Send KOI
        </div>
      </div>
    </button>
  </div>
</div>
`;
